### PR TITLE
Adjust Imports use Package Reference and Cleanup Repo

### LINKED ISSUE #257: Adjust pydvpy import
`pydvpy.py` is importing `curve.py` using a relative import, which can cause import errors when `pydvpy.py` is imported by an external module. This should be changed to an absolute import.

### PR DESCRIPTION
**Pull request by fillmore1**
### Description

Resolves #257

- Reorganizes repo so that only pydv python modules are in the pydv package.
- Changes pydv imports to include package name.
- General code cleanup including:
   - Removing unused imports
   - Removing "star" imports
    - Applying PEP8 style as much as possible

### Checklist:

- [x] I have made corresponding changes to the documentation.
- [x] I have build the docs on sphinx and they look correct.
- [ ] I have updated the release notes.
- [ ] I have run `pydv/scripts/update_version_and_date.py <major, minor, patch>`.

### CODE REVIEW

File: pydv/pdv.py [hunk 1]
```diff
@@ -93,13 +94,14 @@
 import code
 from numbers import Number
 
-import pydvpy as pydvif
```
Comment (rusu24edward) [inline pydv/pdv.py:96]: Please change to `from pydv import pydvpy, curve, pdvplot, pdvutil`

File: pydv/pdvplot.py [hunk 2]
```diff
@@ -59,21 +59,23 @@
...
+from os import path
+
 import matplotlib.pyplot as plt
 from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg
 
-from os import path
-
-from pdvnavbar import PyDVToolbar
-import pdvutil
+import pydv.pdvutil
```
Comment (rusu24edward) [inline pydv/pdvplot.py:67]: Please change to `from pydv import pdvutil`

File: pydv/pydvpy.py [hunk 3]
```diff
@@ -71,37 +71,37 @@
...
 
 try:
     from matplotlib import style
+
     stylesLoaded = True
 except:
     stylesLoaded = False
 
-import curve
+import pydv.curve
```
Comment (rusu24edward) [inline pydv/pydvpy.py:95]: Please change to `from pydv import curve`

**Review by rusu24edward — CHANGES_REQUESTED** [review]
@fillmore1 Thank you for submitting this PR. I've made an initial review with some requested changes. Please also change the target to [pydv-257-adjust-pydv-import](https://github.com/llnl/pydv/tree/pydv-257-adjust-pydv-import). That will allow me to more easily test your changes.

**Comment by fillmore1** [comment]
@rusu24edward Import changes made
Out of curiosity, why `from pydv import _` and not `import pydv._`? Is it just a style preference?

**Comment by rusu24edward** [comment]
I like `from a import b` over `import a.b` because then I can just use `b.function` instead of `a.b.function`.

**Pull request closed without merging**